### PR TITLE
add aliases for services

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | api-platform/api-platform#429
| License       | MIT
| Doc PR        | 

Maybe we should add other services too ?

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
         <service id="api_platform.serializer.context_builder" class="ApiPlatform\Core\Serializer\SerializerContextBuilder" public="false">
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
         </service>
+        <service id="ApiPlatform\Core\Serializer\SerializerContextBuilder" alias="api_platform.serializer.context_builder.filter.inner"/>
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Resources/config/api.xml]: Shouldn't the alias name been the interface?

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml [hunk 2]
```diff
@@ -142,6 +143,7 @@
 
             <tag name="kernel.event_listener" event="kernel.request" method="onKernelRequest" priority="2" />
         </service>
+        <service id="ApiPlatform\Core\EventListener\DeserializeListener" alias="api_platform.listener.request.deserialize"/>
```
Comment (dunglas) [inline src/Bridge/Symfony/Bundle/Resources/config/api.xml]: When can it be necessary to inject this class?

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml (hunk 1 above)
Comment (sroze) [reply src/Bridge/Symfony/Bundle/Resources/config/api.xml]: 👍

Comment (Simperfit) [reply src/Bridge/Symfony/Bundle/Resources/config/api.xml]: Yeah you are right

File: src/Bridge/Symfony/Bundle/Resources/config/api.xml (hunk 2 above)
Comment (Simperfit) [reply src/Bridge/Symfony/Bundle/Resources/config/api.xml]: It does trigger a notice, So we should look

**Comment by norkunas** [comment]
ApiPlatform\Core\Api\IriConverterInterface also should be aliased? I use it in custom normalizer

**Comment by soyuka** [comment]
I think that this has been implemented by #1516, feel free to reopen @Simperfit

**Pull request closed without merging**